backend-project/small_eod/cases/factories_test.py

```diff
@@ -0,0 +1,49 @@
+from django.test import TestCase
```

Comment:
Proponuje przenieść ten plik do ```backend-project/small_eod/cases/tests/factories.py```. Wówczas będzie wszystko dobrze wykryte w domyślnej konfiguracji.

Reply:
Owszem ale tutaj chodzi o konwencję (taka jak w golang) gdzie test jest blisko modułu który testuje, odrazu rzuca się w oczy, moim zdaniem przyjemny w pracy.

Reply:
Owszem ale tutaj chodzi o konwencję (taka jak w golang) gdzie test jest blisko modułu który testuje, odrazu rzuca się w oczy, moim zdaniem przyjemny w pracy.

Reply:
Konwencję są różne w różnych językach. W świecie Python dominuje podejście, aby testy zbierać w pakiety. Przykładowo Django core, Netbox, Apache Airflow, Flask, Python, cpython, django-rest-framework. Myślę, że pewną wartością, którą warto zachować jest funkcjonowanie w ekosystemie Python, gdy przyjęto taką technologie dla tego modułu projektu.

Reply:
Ok, rozumiem, ale w świecie dominuje dużo rzeczy których nikt nie kwestionuje. https://github.com/watchdogpolska/small_eod/pull/182

Reply:
Ja rozumiem, że można wprowadzać innowacje. Ja jednak w tym projekcie w tym module nie chcę wprowadzać żadnych ryzykownych zmian, bo zakładam, że projekt ma być maksymalnie otwarty na nowe osoby i back-end ma służyć Stowarzyszeniu przez dziesięciolecia. Eksperymenty sprawiają, że nowym osobom trzeba wyjaśniać specyfikę projektu, zamiast bazować na jej doświadczeniu o konwencji.

Prawdopodobnie dla stworzenia back-endu Stowarzyszenie drugiej szansy mieć nie będzie (patrzę realnie na zasoby, chociaż może dobra architektura i znacząco nowe potrzeby na to pozwoli). Front-end może za kilka lat się zestarzeje. W integracjach wokół, gdzie możemy ponieść koszt stworzenia projektu od początku nie widzę żadnego problemu, aby eksperymentować (oczywiście świadomie).